### DISCUSSION TITLE
i3-msg error message

### DISCUSSION THREAD

**Discussion by chickenLemonade** [i3/i3#6091] [Q&A] [answered]
I'm trying to write a script which changes the gaps in a specific workspace when a keybind is pressed. To test this out, I typed "i3-msg gaps inner 50px" into my terminal but got a long error message:

ERROR: Your command: gaps inner 50px
ERROR:                                                   ^^^^
ERROR: Expected one of these tokens: 'current', 'all'
[{"success":false,"parse_error":true,"error":"Expected one of these tokens: 'current', 'all'","input":"gaps inner 50px","errorposition":"           ^^^^"}]

Does anyone have suggestions for how to fix this?

**Comment by orestisfl** [comment, marked as answer]
See https://i3wm.org/docs/userguide.html#changing_gaps

Syntax is
```
gaps inner all set 50
```

**Reply by chickenLemonade** [reply]
Thank you!
